### PR TITLE
Create DisplayFilenameAndLayerOnLCD

### PR DESCRIPTION
**Pull request by adecastilho**
This plugin inserts M117 into the g-code so that the filename is displayed on the LCD and updates at each layer to display the current layer

### CODE REVIEW

**Comment by fieldOfView** [comment]
You can get the jobname with `Application.getInstance().getPrintInformation().jobName`

File: plugins/PostProcessingPlugin/scripts/DisplayFilenameAndLayerOnLCD [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Cura PostProcessingPlugin
```
Comment (ChrisTerBeke) [inline plugins/PostProcessingPlugin/scripts/DisplayFilenameAndLayerOnLCD]: This file is missing the .py extension.

**Comment by adecastilho** [comment]
fieldOfView - thanks so much! I knew there had to be a way to access the filename, just hadn't figured it out yet.

File: plugins/PostProcessingPlugin/scripts/DisplayFilenameAndLayerOnLCD.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# Cura PostProcessingPlugin
+# Author:   Amanda de Castilho
+# Date:     August 28, 2018
+
+# Description:  This plugin inserts a line at the start of each layer,
+#               M117 - displays the filename and layer height to the LCD
+#               Alternatively, user can override the filename to display alt text + layer height
+
+..Script import Script
```
Comment (fieldOfView) [inline plugins/PostProcessingPlugin/scripts/DisplayFilenameAndLayerOnLCD.py]: you dropped a `from`

File: plugins/PostProcessingPlugin/scripts/DisplayFilenameAndLayerOnLCD.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+                }
+            }
+        }"""
+    
+    def execute(self, data):
+        if self.getSettingValueByKey("name") != "":
+            name = self.getSettingValueByKey("name")
+        else:
+            name = Application.getInstance().getPrintInformation().jobName       
+        lcd_text = "M117 " + name + " layer: "
```
Comment (fieldOfView) [inline plugins/PostProcessingPlugin/scripts/DisplayFilenameAndLayerOnLCD.py]: could be nice to have the `layer` string translatable

**Pull request merged**